MOXA12XX/iocBoot/iocMOXA12XX-IOC-01/config.xml

```diff
@@ -4,7 +4,7 @@
 <ioc_desc>Moxa ioLogik E1210 Remote I/O</ioc_desc>
```

Comment:
IOC details and description needs to be specific to either the model (10, 40, or, 62). Note this is also not updated for the details in the opi_info.xml (ibex_gui) but not sure if that comment belongs in the 12xx IOC ticket if it exists.